pkg/winupgrade/upgrade.go

```diff
@@ -316,14 +354,6 @@ func verifyPodImageWithHostPathVolume(cs kubernetes.Interface, nodeName string,
 		return false
 	}
 
-	// Get the calico node image
-	nodeDs := daemonset("calico-node")
-	nodeImage, err := nodeDs.getContainerImage(cs, "calico-system", "calico-node")
-	if err != nil {
-		return err
-	}
-	log.Infof("Found node container image is %v\n", nodeImage)
-
 	// Walk through pods
 	for _, pod := range list.Items {
 		if !hasHostPathVolume(pod, hostPath) {
```

Comment:
I'm thinking we should make sure only one pod mounted the host path. So I suggest we get that pod first.
```
var podWithHostPath *v1.pod
count := 0
for _, pod := range list.Items {
    if hasHostPathVolume(pod, hostPath) {
       podWithHostPath = pod
       count ++
    }
}

if count == 0 {
    return fmt.Errorf("Failed to find calico-windows-upgrade pod")
}
if count > 1 {
  return fmt.Errorf("More than one pod have host path")
}

// Continue to check image
...

```